credentials/version.go

```diff
@@ -2,7 +2,7 @@ package credentials
 
 var (
 	// Name is filled at linking time
-	Name = ""
+	Name = "unknown"
```

Comment:
Perhaps we should keep this empty, for cases where someone would "go install"; might be confusing?